src/@next/Pagination/PaginationStyle.tsx

```diff
@@ -58,6 +63,30 @@ export const StyledSimplePaginationButton = styled(StyledPageButton)`
   pointer-events: none;
 `;
 
+export const StyledBorderPaginationButton = styled(StyledPageButton)`
+  border: 1px solid ${Neutral.B68};
+
+  &:first-of-type {
+    border-radius: ${borderRadius2} 0 0 ${borderRadius2};
+  }
+
+  &:last-of-type {
+    border-radius: 0 ${borderRadius2} ${borderRadius2} 0;
+  }
+
+  &:hover {
+    background: ${Blue.S08};
+  }
+
+  &:disabled {
+    background: ${Neutral.B95};
+  }
+`;
+
+export const StyledLabel = styled.div`
+  padding: ${space4};
```

Comment:
label should be vertically centered with 12px of paddings on the left and right
![image](https://github.com/glints-dev/glints-aries/assets/12844836/43b75947-6f3d-4fd5-8c4e-6a989e16890f)